src/jit/flowgraph.cpp

```diff
@@ -23870,6 +23888,253 @@ void Compiler::fgUpdateFinallyTargetFlags()
             }
         }
     }
-
 #endif // FEATURE_EH_FUNCLETS && defined(_TARGET_ARM_)
 }
+
+class FatCalliTransformer
```

Comment:
Please add a comment block explaining at a high-level the input to output transform its does for a calli